cypress/integration/visitorSeesLocationSpecificArticle.spec.js

```diff
@@ -0,0 +1,36 @@
+import stubLocation from "../support/stubLocation";
+
+describe("Visitor can see articles when visiting the App", () => {
+  beforeEach(() => {
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v0/articles",
+      response: "fixture:articles.json"
+    });
+    cy.route({
+      method: "GET",
+      url: "http://localhost:3000/api/v0/articles?**",
+      response: "fixture:local_articles.json"
+    });
+    cy.visit(
+      "http://localhost:3001",
+      stubLocation({ latitude: 40.73061, longitude: 73.935242 })
+    );
+  });
+
+  it("Two articles are displayed", async () => {
```

Comment:
this test is not needed

Reply:
If i removes it next one fails

Reply:
why?

Reply:
Because the whole test was green for wrong reason, WIth the wildcard url fick aldrig testet upp section "local research" , men nu har ja ändrat url, den får upp local reaserach och ser article. So fixed